src/plugins/integrations/AWSIntegration.ts

```diff
@@ -174,7 +178,6 @@ class AWSIntegration implements Integration {
                 [DBTags.DB_STATEMENT_TYPE]: statementType ? statementType : 'READ',
                 [SpanTags.OPERATION_TYPE]: statementType ? statementType : 'READ',
```

Comment:
we might use empty operation type instead of `READ` if the operation is not known

Reply:
Done